index.html

```diff
@@ -173,6 +173,24 @@ <h3>Mauro Duarte</h3>
             </div>
           </div>
 
+          <div class="col-lg-4 col-md-6">
+            <div class="speaker">
+              <img src="img/speakers/larissa_yasin.jpg" alt="Larissa Yasin" class="img-fluid">
+              <div class="details">
+                <h3>Larissa Yasin</h3>
+                <p>
+                  Desenvolvedora Mobile na DBServer. Foco principal no desenvolvimento nativo Android mas as vezes acaba mexendo com iOS e plataformas híbridas. Desenvolveu alguns APPs usando ARCore e ARKit no último ano.
+                  Formada em Sistemas de Informação pela PUCRS e especialização em Desenvolvimento de Aplicações para Dispositivos Móveis pela Unisinos.
```

Comment:
Please, break lines at 80th column, more or less.
Makes it ease to read on editors.